demo/demoitem.cpp

```diff
@@ -15,8 +15,10 @@
  * You should have received a copy of the GNU Lesser General Public License
  * along with this program; if not, see https://www.gnu.org/licenses.
  ****************************************************************************/
-
 #include "demoitem.h"
+#if QT_VERSION > QT_VERSION_CHECK(5, 0, 0)
```

Comment:
QT_VERSION > QT_VERSION_CHECK(5, 0, 0) should be replace with QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)

Reply:
This should be fixed also in all other places

Reply:
Oh duh.  Okay, I think I got them all.   Sorry about that.